src/routes/accounts.ts

```diff
@@ -54,16 +54,15 @@ const requestAccountFields = [
 
 export const showAccountBookings = async (req, res) => {
   const { account_id: accountId } = req.params;
-  const {
-    booking_date: { min: minBookingDate },
-  } = req.query;
+  const minBookingDate = req.query.booking_date?.min;
 
   const person = await findPersonByAccountId(accountId);
 
   const transactions = _.get(person, "transactions", []).filter(
```

Comment:
The db is just an in-memory redis implementation. Although this is out of scope for this PR, I believe the reson for the in-place filter was that redis has no filtering capabilities since it treats the values as opaque.